### PR TITLE
[unit-tests-only] role of collaborator

### PR DESCRIPTION
**Pull request by individual-it**
## Description
test the display of the role of the collaborator

## Related Issue
part of #5234

## How Has This Been Tested?
:robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/ShowCollaborator.spec.js [hunk 1]
```diff
@@ -273,6 +273,39 @@ describe('Collaborator component', () => {
...
+        { name: 'editor', expectedText: 'Editor' },
+        { name: 'advancedRole', expectedText: 'Advanced permissions' },
+        { name: 'owner', expectedText: 'Owner' },
+        { name: 'resharer', expectedText: 'Resharer' },
+        { name: 'invalidRole', expectedText: 'Unknown Role' }
+      ])('shows the role label', cases => {
+        const wrapper = createWrapper({
+          role: { name: cases.name }
+        })
+        expect(wrapper.find('.files-collaborators-collaborator-role').text()).toEqual(
```
Comment (kiranparajuli589) [inline packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/ShowCollaborator.spec.js:288]: you can reuse this selector using as a variable

**Review by saw-jan — COMMENTED** [review]
LGTM

File: packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/ShowCollaborator.spec.js (hunk 1 above)
Comment (individual-it) [reply packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/ShowCollaborator.spec.js]: @kiranparajuli589 done, please review again

**Review by kiranparajuli589 — APPROVED** [review]
lgtm

**Pull request merged**